Source/Scene/Expression.js

```diff
@@ -65,6 +65,16 @@ define([
                 if (defined(val)) {
                     node = new Node(val);
                 }
+            } else if (call === 'rgba') {
```

Comment:
According to that link, those are the only legal css colors.

Yes, you are correct. I will update this.